tests/integration/devfile/cmd_logs_test.go

```diff
@@ -66,6 +67,29 @@ var _ = Describe("odo logs command tests", func() {
 				out = helper.Cmd("odo", "logs", "--deploy").ShouldPass().Out()
 				Expect(out).To(ContainSubstring("no containers running in the specified mode for the component"))
 			})
+			When("--follow flag is specified", func() {
+				var logsSession helper.LogsSession
+				var err error
+
+				BeforeEach(func() {
+					logsSession, _, _, err = helper.StartLogsFollow("--dev")
+					Expect(err).ToNot(HaveOccurred())
+				})
+				AfterEach(func() {
+					logsSession.Kill()
+				})
+				It("should successfully follow logs of running component", func() {
+					var linesOfLogs int
+					Consistently(func() bool {
+						logs := logsSession.OutContents()
+						if len(string(logs)) < linesOfLogs {
+							return false
+						}
+						linesOfLogs = len(logs)
+						return true
+					}, 20*time.Second, 5).Should(BeTrue())
```

Comment:
The way this test and subsequent `--follow` tests are written is that we aren't really bothered about what's being printed on stdout, but we care if the number of lines between successive runs is increasing.

The container image being used is `docker.io/dharmit/infiniteloop` (I am perfectly OK to use a more centralized container image). All it does is print a message on stdout every 5 seconds. Its Dockerfile is:
```Dockerfile
FROM fedora

CMD while true; do echo "`date` - this is infinite for loop"; sleep 1; done
```

So between the checks, we check if there are more lines in the current logs than previous one.

Reply:
> ```dockerfile
> FROM fedora
> 
> CMD while true; do echo "`date` - this is infinite for loop"; sleep 1; done
> ```

I've modified this to:
```dockerfile
FROM alpine

CMD while true; do echo "`date` - this is infinite while loop"; sleep 5; done
```